Automattic/jetpack
# RNA: Scope global CSS modifications

Fixes #26549

#### Changes proposed in this Pull Request:

* Scoped the changes made on global CSS on `IconTooltip` component

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Checkout the branch (not testable on JN)
* Run storybook: `( cd projects/js-packages/storybook && pnpm run storybook:dev )`
* Check that `JS Packages/Components/IconTooltip` does not change (from https://automattic.github.io/jetpack-storybook/?path=/story/js-packages-components-icontooltip--has-content)
* Build VideoPress package: `jetpack build packages/videopress`
* Go to the VideoPress dashboard
* Hover over the quick actions
* Check that the padding is correct

Before | After
---------|----------
![2022-10-03_16-06-21](https://user-images.githubusercontent.com/8486249/193658412-d7d86be0-f033-4657-b5f7-9ac1978ea690.png) | ![2022-10-03_16-05-52](https://user-images.githubusercontent.com/8486249/193658429-20c2f1bc-9be4-4150-9a81-dfc61b70997f.png)
